docs/integrations/sources/stripe.md

```diff
@@ -64,6 +64,7 @@ If you would like to test Airbyte using test data on Stripe, `sk_test_` and `rk_
 
 | Version | Date       | Pull Request | Subject |
 | :------ | :--------  | :-----       | :------ |
+| 0.1.14   | 2021-07-09 | [4669](https://github.com/airbytehq/airbyte/pull/4669) | Add status param for Subscriptions Stream |
```

Comment:
Change PR description and docs description to :
Stripe Source: retrieve all kinds of Subscriptions (including expired and canceled)

Reply:
Done.